This patch fixes compilation of printer support units on architectures which do
not have libc unit (ie all except i386).

Index: components/Makefile.fpc
===================================================================
--- lazarus~/components/Makefile.fpc	(révision 15506)
+++ lazarus/components/Makefile.fpc	(copie de travail)
@@ -4,7 +4,7 @@
 #
 
 [target]
-dirs=synedit codetools lazcontrols
+dirs=synedit codetools lazcontrols printers printers/design fpcunit fpcunit/ide memds projecttemplates rtticontrols sdf sqldb tachart tdbf turbopower_ipro jcf2/IdePlugin/lazarus todolist chmhelp/packages/help chmhelp/packages/idehelp
 
 [clean]
 files=$(wildcard ./units/*$(OEXT)) \
Index: components/turbopower_ipro/Makefile.fpc
===================================================================
--- lazarus~/components/turbopower_ipro/Makefile.fpc	(révision 17880)
+++ lazarus/components/turbopower_ipro/Makefile.fpc	(copie de travail)
@@ -16,6 +16,7 @@
 
 [target]
 units=turbopoweripro.pas
+dirs=design
 
 [clean]
 files=$(wildcard $(COMPILER_UNITTARGETDIR)/*$(OEXT)) \
